src/ar.erl

```diff
@@ -86,7 +89,8 @@
 	internal_api_secret = not_set,
 	enable = [],
 	disable = [],
-	content_policies = []
+	content_policies = [],
+	transaction_blacklist = []
```

Comment:
Looks like this is a list of filenames. I think it makes sense to name it in plural like `content_policies`.

For the record, I would rather call them `content_policy_files` and `transaction_blacklist_files`.

Reply:
✅